src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -156,16 +165,24 @@ class ExternalIssueForm extends AsyncComponent {
         submitLabel={SUBMIT_LABEL_BY_ACTION[action]}
         footerClass="modal-footer"
       >
-        {config.map(field => (
-          <FieldFromConfig
-            key={field.name}
-            field={field}
-            inline={false}
-            stacked
-            flexibleControlStateSize
-            {...this.getFieldProps(field)}
-          />
-        ))}
+        <Accordion
+          cutoff={3}
+          onClick={() => this.setState({showAllFields: !this.state.showAllFields})}
+          open={this.state.showAllFields}
+          style={{margin: '1em 0'}}
+          label={this.state.showAllFields ? 'Show Less Fields' : 'Show More Fields'}
```

Comment:
We'll probably want two props for these labels

Reply:
Not sure I understand.

Reply:
e.g. `labelMore` and `labelLess` since we're moving "show all" state into the Accordion component.